hazelcast-spring/src/main/resources/hazelcast-spring-3.8.xsd

```diff
@@ -314,13 +460,14 @@
                                     <xs:element name="partition-strategy" type="xs:string" minOccurs="0" maxOccurs="1"/>
                                 </xs:sequence>
                                 <xs:attribute name="name" use="required" type="xs:string"/>
-                                <xs:attribute name="in-memory-format" use="optional" type="xs:string" default="BINARY">
+                                <xs:attribute name="in-memory-format" use="optional" type="in-memory-format" default="BINARY">
```

Comment:
That is the exception people would see in the conditions i explained above but I just wanted to have double checked by someone. I might also be too skeptical about it :)